doc/Writerside/topics/first_data_flow.md

```
@@ -0,0 +1,28 @@
+1. Step with a code block
+
+   ```bash
+    run this --that
+   ```
+
+2. Step with a [link](https://www.jetbrains.com)
+
+3. Step with a list.
+   - List item
+   - List item
+   - List item
```

Markdownlint

Files should end with a single newline character

(MD047, single-trailing-newline)